dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/preheat/TrackerPreheat.java

```diff
@@ -175,13 +175,13 @@
     private Map<TrackerIdScheme, Map<String, TrackedEntityComment>> notes = new EnumMap<>( TrackerIdScheme.class );
 
     /**
-     * A Map of event uid and preheated {@see ProgramInstance}. The value is a
-     * List, because the system may return multiple ProgramInstance, which will
-     * be detected by validation
+     * A Map of trackedEntity uid and preheated {@see ProgramInstance}. The
```

Comment:
This comment is not relevant anymore.
A trackedEntity can be linked to more than one ProgramInstance, there is no validation that will detect it